index.html

```diff
@@ -77,7 +77,7 @@ <h1 class="feature-header">
           <h1 class="feature-header">
             Record once, debug anywhere
           </h1>
-          <p>  Web Replay was made for collaboration, so it's possible to save and replay recordings. Stash a recording and pick up where you left later, or share the recording with somebody else.</p>
+          <p> Web Replay is built with collaboration in mind. Reporting issues is as simple as, recording the bug, publishing and sharing a URL. </p>
```

Comment:
There is a stray comma here.  How about "Reporting issues is as simple as recording the bug, saving the recording to the cloud, and sharing its URL"